PanCardView/CardsView.cs

```diff
@@ -1731,7 +1732,7 @@ private void ExecutePreventInvalidOperationException(Action action, int restartC
                             ExecutePreventInvalidOperationException(action, ++restartCount);
                             return;
                         }
-                        Console.WriteLine("CardsView: Couldn't handle InvalidOperationException");
+                        Debug.WriteLine("CardsView: Couldn't handle InvalidOperationException");
```

Comment:
Can we preserve ```Console.Writeline``` and remove ```using System.Diagnostics;```

Reply:
No, We can't. Console is not supported in net.standard1.0

Reply:
System.Console package is available only in net.standard1.3 or higher.

Reply:
I can save `Console` for net.standard 2.0 if you want. But I can't see a big reason for it.